homework/style.css

```diff
@@ -1,3 +1,70 @@
+/* cSpell:disable */
+
 .alert-error {
-    color: red;
-}
\ No newline at end of file
+  color: red;
```

Comment:
Hi Nadin - I think the styling of the page can be a bit better as currently, contributors list is appended right after the repository information and it doesn't look nice. Could you please have a look?